### ISSUE TITLE
Let business be business

### ISSUE THREAD

**Issue by greatbody** [stop-war-in-ukraine/stop-russia-it#42]
If JetBrains can revoke license to developers in Russia, I can use cracked software as well.

**Comment by digiom-ceo** [comment]
Especially when a Russian from Moscow revived PHP and PHPstorm.

**Comment by oleksiizabara** [comment]
да! можете! но ни один коиент из цивилизованной страны не будет работать с вами! 
хотя они и так не будут с вами работать больше дае если вас не лишат лицензии! 

так что хватить хвататься за соломинки! приезжайте в Украину и продолжайте вести честный бизнес без ограничений со всем миром! в фашисткой россии разработчикам больше ловить нечего

**Comment by digiom-ceo** [comment]
> да! можете! но ни один коиент из цивилизованной страны не будет работать с вами!

Во сне приснилось? С каких пор цивилизованные страны стали фашистами вроде вас? Вас всего пару миллионов в хохляндии и мы освободим Украину от нацизма.

**Comment by digiom-ceo** [comment]
Ukrainian Nazis delete comments. There is no democracy.

**Comment by greatbody** [comment]
> да! можете! но ни один коиент из цивилизованной страны не будет работать с вами! хотя они и так не будут с вами работать больше дае если вас не лишат лицензии!
> 
> так что хватить хвататься за соломинки! приезжайте в Украину и продолжайте вести честный бизнес без ограничений со всем миром! в фашисткой россии разработчикам больше ловить нечего

If JetBrains can revoke license to developers in Russia, and it is considered as political action, thus we can take our cracked software policy also as an political action, which means when the war ends, we will still do business.

Otherwise, that will be double standard.

But looks like the westen world love double standard. But we workers don't.

Workers in the world, United!
